### ISSUE TITLE
Keyboard focus should be moved to opened drawer

### ISSUE THREAD

**Issue by jianliao** [directus/directus#5344]
1) When you open a drawer, the keyboard focus will not automatically move to the drawer, it will stay on body element while it should move to the first non-disabled input element on open drawer.
2)

https://user-images.githubusercontent.com/1207520/116490379-13e9c380-a84c-11eb-87e1-305e3b03dc0d.mp4

3) Latest version

**Comment by jianliao** [comment]
I try directive  `v-focus` on the drawer element, but it does not work.

**Comment by jianliao** [comment]
Above PR is not that perfect and just to set the ball rolling.

**Comment by rijkvanzanten** [comment]
Nice! Thanks for that first starting point @jianliao

We'll have to do a bigger keyboard accessibility sprint sometime in the near future to polish up some of these interactions

**Comment by jianliao** [comment]
Thanks, @rijkvanzanten. Looking forward to the unified solution. This issue was discovered by one of the spectrum designers who need to create a lot of new content. Whenever she click `Create New`, she wishes to start inputting with keyboard immedietaly.

**Comment by rijkvanzanten** [comment]
Yeah that makes total sense
